app/controllers/api/v1/contacts_controller.rb

```diff
@@ -23,6 +23,13 @@ def update
     end
   end
 
+  def destroy
+    Contact.destroy(params[:id])
+    render nothing: true, status: 204
+  rescue ActiveRecord::RecordNotFound
```

Comment:
This is using exceptions for control flow. This is not really an "exceptional case" you expect it often enough to write a spec around it.